provision/service.go

```diff
@@ -81,6 +84,14 @@ func New(cfg Config, sdk SDK.SDK, logger logger.Logger) Service {
 	}
 }
 
+// Mapping retrieves current configuration
+func (ps *provisionService) Mapping(token string) (interface{}, error) {
```

Comment:
Maybe we can at least return `map[string]interface` instead of interface{}. It's of little use, but better than `interface{}`.